common/credsregistry/fields.go

```diff
@@ -0,0 +1,157 @@
+package credsregistry
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/amp-labs/connectors/common/handy"
+	"github.com/amp-labs/connectors/providers"
+	"github.com/iancoleman/strcase"
+)
+
+// Fields is a grouping of constant values that dictate the keys that
+// can be present inside *creds.json file.
+var Fields = struct { // nolint:gochecknoglobals
+	Provider Field
```

Comment:
These are constant values that define consistent naming of ENV variables and JSON fields